cmd/gearcmd/main.go

```diff
@@ -25,6 +27,7 @@ func main() {
 	cmdTimeout := flag.Duration("cmdtimeout", 0, "Maximum time for the command to run before it will be killed, e.g. 2h, 30m, 2h30m")
 	retryCount := flag.Int("retry", 0, "Number of times to retry the job if it fails")
 	warningLength := flag.Int("warningLength", 5, "Number of warning lines to store and send back to the gearmn job")
+	passSigterm := flag.Bool("pass-sigterm", false, "Whether or not to pass SIGTERM through to the worker process")
```

Comment:
If we change it to always pass SIGTERM through, anything that doesn't have a SIGTERM listener will immediately die instead of maintaining current behavior.